src/transforms/aggregate.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
 /// Configuration for the `aggregate` transform.
 #[configurable_component(transform("aggregate"))]
 #[derive(Clone, Debug, Default)]
-#[serde(deny_unknown_fields, default)]
+#[serde(deny_unknown_fields)]
```

Comment:
Telling `serde` to use a default value for deserializing the configuration of a typed component makes no sense, because these types are never a fixed field... only part of a field using their containing enums i.e. `Transforms`.

It does, however, lead to the `vector-config-macros` code thinking that there's a default value that should be added to the configuration schema, which again... does not make sense at _this_ particular level.